Homework_9/Task_1.py

```diff
@@ -0,0 +1,5 @@
+with open('test_txt', 'r') as test_txt:
+    text = test_txt.read()
+with open('new_test.txt', 'w') as new_test:
+    new_test.write(text)
+print("Text copied successfully to new_test.txt")
```

Comment:
I am not sure that we need print() here.
Your code can be more minimalistic
![Uploading image.png…]()